### PR TITLE
base template

### PR DESCRIPTION
**Pull request by malisara**

### CODE REVIEW

File: sir_cake/seller/templates/seller/base.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                {% endblock %}
+            </div>
+        </div>
+    </div>
+
+    <!--JS Bundle -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js"
```
Comment (inejc) [inline sir_cake/seller/templates/seller/base.html]: What is popper?

File: sir_cake/seller/templates/seller/base.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js"
+        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+        crossorigin="anonymous"></script>
+    <!--CHART.JS-->
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```
Comment (inejc) [inline sir_cake/seller/templates/seller/base.html]: I think you will only need `chart.js` on a single page? If so, it makes sense to only include it there

File: sir_cake/sir_cake/settings.py [hunk 3]
```diff
@@ -11,6 +11,8 @@
 """
 
 from pathlib import Path
+import os
+from django.contrib.messages import constants as messages
```
Comment (inejc) [inline sir_cake/sir_cake/settings.py:15]: Is this import needed?

File: sir_cake/sir_cake/settings.py [hunk 4]
```diff
@@ -36,6 +38,10 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    'crispy_forms',
+
+    'seller.apps.SellerConfig',
```
Comment (inejc) [inline sir_cake/sir_cake/settings.py]: Please remove white spaces here :P

Comment (inejc) [inline sir_cake/sir_cake/settings.py]: If you decided to add seller app related stuff, please also add all of the seller app files. I believe `SellerConfig` is missing right now which means as soon as we merge this PR the main branch will be in a broken state.

**Review by inejc — CHANGES_REQUESTED** [review]

File: sir_cake/sir_cake/settings.py (hunk 3 above)
Comment (malisara) [reply sir_cake/sir_cake/settings.py:15]: It's for changing Django default message tag (error to danger) to work with Bootstrap alerts :)

**Review by inejc — APPROVED** [review]

**Pull request merged**